test/integration/alpn_integration_test.cc

```diff
@@ -136,6 +136,34 @@ TEST_P(AlpnIntegrationTest, Http1New) {
   codec_client2->close();
 }
 
+TEST_P(AlpnIntegrationTest, Http1RememberLimits) {
```

Comment:
The non-H1 cases are covered in the mixed_conn_pool_test, right?

Reply:
and http2 new and old